frog/imports/api/users.js

```diff
@@ -11,7 +11,7 @@ type MeteorUser = {
   emails?: EmailT[],
   username: string,
   isAnonymous: boolean,
-  profile?: { displayName: string }
+  profile?: { displayName: string, isAdmin?: boolean }
```

Comment:
Don't use profile for this: 
https://guide.meteor.com/accounts.html#dont-use-profile